### ISSUE TITLE
🧐 Make sider in ProTable not collapsible

### ISSUE THREAD

**Issue by iloveip** [ant-design/pro-components#976]
Hi there,

Is it possible to make Sider in ProTable not collapsible? There's such API in Sider in Ant Design, where you can pass `collapsible={false}`. But I didn't find the same here.

**Comment by chenshuai2144** [comment]
`search ={ { collapsed:true } } `

code like this

**Comment by iloveip** [comment]
I'm sorry, I meant sider in Pro Layout, not in ProTable.

**Comment by chenshuai2144** [comment]
collapsed | 控制菜单的收起和展开 | boolean | -
onCollapse | 菜单的折叠收起事件 | (collapsed: boolean) => void

**Comment by iloveip** [comment]
Thank you. I tried using the following in my settings:

```
defaultCollapsed={false}
collapsed={false}
breakpoint={false}```

The Sider is not collapsed and disappears on `md` breakpoint, if I shrink the window. But if I make the window larger, it appears again in collapsed mode. Also there is no way to turn off `collapsedButton`. If I set it to `collapsedButtonRender={null}`, then the trigger for the Drawer on mobile also disappears.

It would be better to have a separate prop to control this behavior. For example, if you set it to `collapsible={false}`, then it doesn't get collapsed on window change and no trigger appears for the Sider. But it doesn't effect the trigger for the Drawer.
